e2e/main.py

```diff
@@ -45,7 +45,18 @@ def assert_kit_balance(checker: ContractInterface, address: str, expected_kit: i
     # TODO: There might be a way to get this from contract metadata
     kit_token_id = 0
     kit_balance = checker.metadata.getBalance((address, kit_token_id)).storage_view()
-    assert expected_kit == kit_balance
+    if expected_kit != kit_balance:
+        raise AssertionError(f"Expected {expected_kit} but got {kit_balance}")
+
+
+def assert_wrapped_tez_balance(wrapper: ContractInterface, address: str, expected: int):
+    fa2_balance_of = {
+        "requests": [{"owner": address, "token_id": 2}],
```

Comment:
Ah, this token id should also come from `tezWrapper.ml` actually (or, even better, from `checker.yml`). Let's discuss this a little more before merging, because we need access to the `tez_token_id` in #269 as well. It might be best to open a small PR first, where this field moves to `checker.yaml` and we also bring it to the python world similarly to `kit_token_id`, and then rebase this PR and #269 on top of it. I'm afraid that one of the values will get out of sync (though #269 is probably much more in risk than this pr).